etc/ci.sh

```diff
@@ -37,18 +31,62 @@ else
         make
         cd ../..
 
-        sh bin/gap.sh --cover coverage tst/${TEST_SUITE}.g
+        sh bin/gap.sh -q tst/extractmanuals.g
 
-        # generate coverage report
-        sh bin/gap.sh -q <<GAPInput
-            if LoadPackage("profiling") <> true then
-                Print("ERROR: could not load profiling package");
-                FORCE_QUIT_GAP(1);
-            fi;
-            OutputJsonCoverage("coverage", "coverage.json");
+        for ch in tst/testmanuals/*.tst
+        do
+            COVNAME="coverage.`basename $ch .tst`"
+            sh bin/gap.sh -q --cover $COVNAME <<GAPInput
```

Comment:
On my system with a new SSD, it takes 1.5 seconds to start GAP. Even if it's fully cached in RAM. I do not know how long it takes on Travis, but I wouldn't be surprised if it was slower there. We do this 80 times or so, hence just starting GAP that often costs us 2 minutes or more.

So I would really consider using a workspace here, like we do in the `make` targets.